server/sonar-web/src/main/js/api/report.ts

```diff
@@ -0,0 +1,48 @@
+/*
+* SonarQube
+* Copyright (C) 2009-2016 SonarSource SA
+* mailto:contact AT sonarsource DOT com
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import { getJSON, post } from '../helpers/request';
+
+export interface ReportStatus {
+  canDownload?: boolean;
+  canSubscribe: boolean;
+  componentFrequency?: string;
+  globalFrequency: string;
+  subscribed?: boolean;
+}
+
+export function getReportStatus(component: string): Promise<ReportStatus> {
+  return getJSON('/api/governance_reports/status', { componentKey: component });
+}
+
+export function getReportUrl(component: string): string {
```

Comment:
I'd move it to `helpers/urls`

Reply:
I prefer to keep it isolated in `report.ts`, because this is coming with the governance plugin.

Reply:
Ok it make sense indeed